### DISCUSSION TITLE
Getting feedback on pre-proposal for “Project 1: Classification using EEG”

### DISCUSSION THREAD

**Discussion by senu-nk** [alan-turing-institute/sktime#761] [General]
Hi @TonyBagnall, @mloning

I am interested in applying for “Project 1: Classification using EEG”. I have had some time to think about the project and I made a pre-proposal Google doc with a proposed solution based on my understanding.

I have a few questions on the project (comments in the Google doc), and I was wondering if a project mentor could take a look at my document and answer my questions and provide any feedback (please let me know your email address so I can give you access to the doc).

I will submit a full application later via the Google form on the sktime GSoC page. For now, I thought it would be a good idea to get early feedback on my understanding of the project.
Thank you!

**Comment by mloning** [comment]
Hi @D3ViL-NK, in general we're happy to take a look - probably best if you send us a link via private chat on Gitter to a collaborative document (e.g. https://hackmd.io or Google docs) so that we can leave some comments.

**Reply by fkiraly** [reply]
I'd recommend not to post proposals publicly - just send us the link via private chat
